src/openrct2/drawing/Font.h

```diff
@@ -33,8 +33,8 @@ enum class FontSpriteBase : int16_t
 
 #ifndef NO_TTF
 
-struct _TTF_Font;
-using TTF_Font = _TTF_Font;
+struct Internal_TTF_Font;
```

Comment:
This is the wrong style. Should be `InternalTTFFont`

Reply:
Should be fixed by https://github.com/OpenRCT2/OpenRCT2/pull/15521/commits/fda4476288d28fca3b4b0b00e6a874ca4920bd29 now.